lib/src/layout/tab_view/tab.dart

```diff
@@ -16,61 +16,67 @@ class ZenitTab extends StatelessWidget {
     this.onPressed,
   });
 
-  final String? title;
+  final String title;
   final Widget? icon;
 
   final IconData? closeIcon;
 
   final VoidCallback? onClose;
   final VoidCallback? onPressed;
 
-  final bool enabled;
+  final bool selected;
 
   @override
   Widget build(BuildContext context) {
-    return ActionBase(
-      onTapDown: onPressed,
-      onTertiaryTapDown: (details) => onClose?.call(),
-      margin: const EdgeInsets.symmetric(horizontal: 4),
-      child: SizedBox(
-        height: 36,
-        width: 184,
-        child: DecoratedBox(
-          decoration: BoxDecoration(
-            borderRadius: kDefaultBorderRadiusMedium,
-            color: enabled ? ZenitTheme.of(context).surfaceColor : Colors.transparent,
-          ),
-          child: Padding(
-            padding: const EdgeInsets.symmetric(vertical: 4),
-            child: Row(
-              children: [
-                if (icon != null)
-                  Padding(
-                    padding: const EdgeInsets.fromLTRB(12, 4, 8, 4),
-                    child: icon,
-                  ),
-                Text(
-                  title ?? "Tab",
-                  style: TextStyle(color: ZenitTheme.of(context).foregroundColor),
-                ),
-                const Spacer(),
-                Padding(
-                  padding: const EdgeInsets.fromLTRB(8, 0, 8, 0),
-                  child: InkWell(
-                    borderRadius: kDefaultBorderRadiusSmall,
-                    onTap: onClose,
-                    hoverColor: Colors.red,
-                    child: Padding(
-                      padding: const EdgeInsets.all(4.0),
-                      child: Icon(
-                        closeIcon ?? Icons.close,
-                        size: 14,
-                        color: ZenitTheme.of(context).foregroundColor,
+    // Colors
+    final backgroundColor = selected ? ZenitTheme.of(context).surfaceColor : Colors.transparent;
+    final borderColor = selected
+        ? ZenitTheme.of(context).foregroundColor.withOpacity(0.15)
+        : ZenitTheme.of(context).surfaceColor;
+
+    return SizedBox(
+      width: 200,
+      child: GestureDetector(
+        onTap: onPressed,
+        onTertiaryTapUp: (_) => onClose?.call(),
+        child: MouseRegion(
+          cursor: SystemMouseCursors.click,
+          child: Tooltip(
+            message: title,
+            child: Material(
+              color: backgroundColor,
+              shape: RoundedRectangleBorder(
+                borderRadius: kDefaultBorderRadiusMedium,
+                side: BorderSide(color: borderColor),
+              ),
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 8.0),
+                child: Row(
+                  children: [
+                    if (icon != null) SizedBox.square(dimension: 20, child: icon),
+                    const Gap(8),
```

Comment:
i'd recommend dropping the dependency on gap and just using SizedBoxes for gaps like this

Reply:
I know, you've recommended that a few times already. I'm gonna do it eventually - just not now.